osquery/database/database.cpp

```diff
@@ -35,6 +42,11 @@ const std::string kEvents = "events";
 const std::string kCarves = "carves";
 const std::string kLogs = "logs";
 
+const std::string kDbEpochSuffix = "epoch";
+const std::string kDbCounterSuffix = "counter";
+
+const std::string kDatabseResultsVersion = "1";
```

Comment:
Nit: I would say the current is version 1, and this should be version 2?

Reply:
Sure, current version though doesn't exist. There's no value in the DB as it is, so imo current version would be 0?